tests/data_context/test_data_context.py

```diff
@@ -1360,3 +1357,189 @@ def test_get_checkpoint_raises_error_on_missing_batch_kwargs(empty_data_context)
 
     with pytest.raises(CheckpointError) as e:
         context.get_checkpoint("foo")
+
```

Comment:
Because data context has become so large, we need to split the tests into multiple files for clarity. Please move the data docs related tests into a new file, `test_data_context_data_docs_api` that can focus specifically on that behavior and relevant fixtures.

Reply:
Fixed.